.eslintrc.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   extends: '@cmsgov/eslint-config-design-system',
+  parser: 'babel-eslint',
```

Comment:
@nichia what is this for?

Reply:
The [eslint parsing error for `static` syntax](https://stackoverflow.com/questions/36001552/eslint-parsing-error-unexpected-token) and added `babel-eslint` to fix that.

Also `static` requires  `@babel/plugin-proposal-class-properties`